src/main/java/com/google/api/codegen/config/PageStreamingConfig.java

```diff
@@ -246,8 +246,15 @@ static PageStreamingConfig createPageStreamingConfig(
     if (pageSizeField == null) {
       // TODO: Conform to design doc spec, once approved, for using non-standard paging fields
       //       (such as max_results for page_size)
-      if (language == TargetLanguage.JAVA && transportProtocol == TransportProtocol.HTTP) {
-        pageSizeField = methodModel.getInputField(pagingParams.getNameForMaxResults());
+      if (transportProtocol == TransportProtocol.HTTP) {
+        if (language == TargetLanguage.JAVA) {
+          pageSizeField = methodModel.getInputField(pagingParams.getNameForMaxResults());
+        } else if (language == TargetLanguage.PHP) {
+          FieldModel resourcesField = ProtoPageStreamingTransformer.getResourcesField(methodModel);
+          if (resourcesField != null && !resourcesField.isMap()) {
```

Comment:
Do we not need this in Java because Java *is* handling map responses?

Reply:
That is my understanding, yes!